src/js/graphql_endpoint/modules/dgraph_client.ts

```diff
@@ -0,0 +1,23 @@
+import dgraph = require("dgraph-js");
+import grpc = require("@grpc/grpc-js");
+
+const get_random = <T>(list: T[]): T => {
+	return list[Math.floor(Math.random() * list.length)];
+};
+
+const mg_alpha = get_random(process.env.MG_ALPHAS.split(","));
+
+export type DgraphClient = dgraph.DgraphClient;
+export const getDgraphClient = (): dgraph.DgraphClient => {
+	const clientStub = new dgraph.DgraphClientStub(
+		mg_alpha,
+		grpc.credentials.createInsecure()
+	);
+
+	return new dgraph.DgraphClient(clientStub);
+};
+
+export interface RawNode {
```

Comment:
Not sure if RawNode also needs `nodeKey: string`?

Reply:
https://github.com/grapl-security/issue-tracker/issues/337
probably not :)